packages/flutter/lib/src/material/text_field.dart

```diff
@@ -168,6 +185,28 @@ class TextField extends StatefulWidget {
   /// null, the value must be greater than zero.
   final int maxLines;
 
+  /// The maximum number of characters to allow in the text field.
+  ///
+  /// If set, this will display a character counter below the field, showing
```

Comment:
[just a suggestion] If true, a character counter will be displayed below the field. The character counter shows how many ...